hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/processes/InStreamHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.molgenis.hadoop.pipeline.application.processes;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+/**
+ * Handles the streams retrieved from the process.
+ */
+public class InStreamHandler extends StreamHandler
+{
+	/**
+	 * Stores the write stream to be used.
+	 */
+	InputStream processStream;
+
+	/**
+	 * Object to which the output stream will be written to.
+	 */
+	InContainer inContainer;
+
```

Comment:
Future push already has them as `protected` as this class has been turned into an `abstract class` with subclasses (see application design sent by email).